app/components/event-table.js

```diff
@@ -0,0 +1,12 @@
+import Ember from 'ember';
+
+const { Component } = Ember;
+
+export default Component.extend({
+  didInsertElement() {
+    this._super.call(this);
+    this.$('.button').popup({
```

Comment:
Instead of this, use https://semantic-org.github.io/Semantic-UI-Ember/#/modules/popup for simple popups.